server/graphql/v2/object/Update.js

```diff
@@ -67,6 +69,29 @@ const Update = new GraphQLObjectType({
           return req.loaders.Collective.byId.load(update.CollectiveId);
         },
       },
+      comments: {
+        type: CommentCollection,
```

Comment:
Non null (we return an empty list if no entry)

Reply:
I ended up making it nullable in case user don't have permissions to see the comments